ccmlib/node.py

```diff
@@ -1277,7 +1279,14 @@ def stress_process(self, stress_options, **kwargs):
                 # specify used jmx port if not already set
                 if not [opt for opt in stress_options if opt.startswith('jmx=')]:
                     stress_options.extend(['-port', 'jmx=' + self.jmx_port])
-
+        if self.cluster.tablets_enabled:
+            smp = sum([node._smp or 1 for node in self.cluster.nodes.values()])
+            # default initial tablets: 32 * `sum of SMPs in cluster` coerced to power of 2 (suitable for small number of tables)
+            initial_tablets = int(math.pow(2, math.ceil(math.log(smp*32, 2))))
+            self.debug("tablets enabed, adjusting stress options by replication strategy change.")
+            stress_options = self._set_keyspace_initial_tablets(
+                stress_options, datacenter=self.data_center, rf=1, initial_tablets=initial_tablets
```

Comment:
Why rf=1? seems unrelated here.

Reply:
It's just used to setup a default which matches cassandra stress default which if you don't define the schema option it would be SimpleStrategy with rf=1

I agree it's a bit weird to have it in the arguments if it's a default.

Reply:
updated